scripts/install_ansible_debops_controller.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+################################################################################
+################################################################################
+#
+# https://docs.debops.org/en/master/user-guide/install.html
+#
+# This scripts installs DebOps and its dependencies on a (fresh) system.
+# It creates a dedicated DebOps user (e.g. ansible) who should then execute all DebOps-playbooks.
+# This script only installs DebOps globally , it does not initiliaze any DebOps-project.
+#
+################################################################################
+
+set -o nounset -o pipefail -o errexit
+
+# The username used by debops
+DEBOPS_ADMIN_USER="${DEBOPS_ADMIN_USER:-ansible}"
+
+echo "Install dependencies"
+apt-get -q update
+DEBIAN_FRONTEND=noninteractive apt-get \
+  --no-install-recommends -yq install \
+  iproute2 \
+  levee \
+  openssh-client \
+  python3-apt \
+  python3-distro \
+  python3-dnspython \
+  python3-future \
+  python3-ldap \
+  python3-openssl \
+  python3-pip \
+  python3-wheel \
+  python3-setuptools \
+  procps \
+  sudo \
+  tree \
+  git \
+  gpgv2 \
+  uuid-runtime \
+  sshpass \
+  encfs \
+  dirmngr \
+  rsync \
+  pass
+
+echo "Install ansible and debops"
+pip3 install \
+ debops[ansible]
+
+echo "Cleaning up cache directories..."
+apt clean
+rm -rf /var/lib/apt/lists/* /var/cache/apt/archives/*.deb /root/.cache/*
+
+echo "Add '${DEBOPS_ADMIN_USER}' admin user"
+groupadd --system admins
+echo "%admins ALL = (ALL:ALL) NOPASSWD: SETENV: ALL" > /etc/sudoers.d/admins
+chmod 0440 /etc/sudoers.d/admins
+useradd --user-group --create-home --shell /bin/bash \
+       --home-dir /home/${DEBOPS_ADMIN_USER} --groups admins \
+       --comment '${DEBOPS_ADMIN_USER} Admin User' ${DEBOPS_ADMIN_USER}
```

Comment:
Most users probably already have their own unprivileged account, which could be used instead of creating a separate `ansible` account. This also helps on the remote hosts since DebOps can use the `$USER` value from the Ansible Controller and have the user have his own account created there as well.

Reply:
I added this in the assumption you have only created/got a fresh VM from your provider (e.g. Hetzner), there is only a 'root' (or 'debian' or 'ubuntu') account with my SSH keys. And that's it.

The script takes care of installing ansible+debops and the user for debops. Which as default is called ansible in order to separate users, specially in a multi-admin-user environment. Then there is the issue sharing the `ansible/secret` folder with several other users without getting inconsistencies.

Reply:
Very well, I suppose that is a valid use case. However it would be good to explain the intended use case at the beginning of the script to make it a bit more clear what it is meant for.